src/oscar/apps/checkout/applicator.py

```diff
@@ -0,0 +1,55 @@
+class SurchargeList(list):
+    @property
+    def total(self):
+        return sum([surcharge.price for surcharge in self])
+
+
+class SurchargePrice(object):
```

Comment:
We don't need to support old-style classes any more, having dropped Python 2 support  - i.e., this should just be `class SurchargePrice():`